alacritty/src/display/content.rs

```diff
@@ -239,6 +239,9 @@ impl RenderableCell {
             let config_fg = colors.selection.foreground;
             let config_bg = colors.selection.background;
             Self::compute_cell_rgb(&mut fg, &mut bg, &mut bg_alpha, config_fg, config_bg);
+            if content.config.colors.transparent_background_colors {
```

Comment:
I went with the simplest solution. The alternative I thought of was to add `transparent_background_colors` as an optional argument to the `compute_cell_rgb`, and we change the `compute_cell_rgb` to something like this:

```rust
    fn compute_cell_rgb(
        cell_fg: &mut Rgb,
        cell_bg: &mut Rgb,
        bg_alpha: &mut f32,
        fg: CellRgb,
        bg: CellRgb,
        transparent_background_colors: bool
    ) {
        let old_fg = mem::replace(cell_fg, fg.color(*cell_fg, *cell_bg));
        *cell_bg = bg.color(old_fg, *cell_bg);

        if !transparent_background_colors && bg != CellRgb::CellBackground {
            *bg_alpha = 1.0;
        }
    }
```

And while calling this on Line 241 here we do:
```rust
Self::compute_cell_rgb(&mut fg, &mut bg, &mut bg_alpha, config_fg, config_bg, true);
```